rag-langchain/README.md

```diff
@@ -30,5 +30,5 @@ snapshot_download(repo_id="BAAI/bge-base-en-v1.5",
 Follow the instructions below to build you container image and run it locally. 
 
 * `podman build -t ragapp rag-langchain -f rag-langchain/builds/Containerfile`
-* `podman run --rm -it -p 8501:8501 -v Local/path/to/locallm/models/:/rag/models:Z -v Local/path/to/locallm/data:/rag/data:Z ragapp -- -H 10.88.0.1 -m http://10.88.0.1:8001/v1`
+* `podman run --rm -it -p 8501:8501 -v Local/path/to/locallm/data:/rag/data:Z ragapp -- -H 10.88.0.1 -m http://10.88.0.1:8001/v1`
```

Comment:
without the `models/` volume mount you will download the embedding model at runtime. I think we want to avoid downloading models at runtime.

Reply:
I don't get the role of the model service then. Isn't this going to first look at the vector db and then call the model service which has the model mounted?

Reply:
It is a separate model for creating the vector embeddings. RAG apps need two models: a language model, which is in the model service and an embedding model in the client to convert input text to vector for the vector db.